Dockerfile

```diff
@@ -0,0 +1,23 @@
+FROM python:3.13-alpine3.21
+LABEL maintainer="dmysamo@gmail.com"
+
+ENV PYTHONUNBUFFERED 1
+
+WORKDIR app/
```

Comment:
The `WORKDIR` should be specified as an absolute path. Changing it to `/app` will align with best practices.